unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -913,16 +937,12 @@ Do not provoke the tender heart of the Lamb against you,
 who also hath the voice of a lion,
 and can roar terribly out of his holy mountain against the enemies of his life and Spirit.
 
-=== A NECESSARY WARNING
-
-And of very great Importance to all that call themselves CHRISTIANS,
-and hope for a Share in the Book of LIFE, and the escaping the Damnation of Hell;
-which is their Portion whose Names are written in the Book of Death,
-and blotted by God out of the Book of LIFE, though they hope to find them written there.
+[.old-style]
+=== A Necessary Warning / And of very great Importance to all that call themselves CHRISTIANS, and hope for a Share in the Book of LIFE, and the escaping the Damnation of Hell; which is their Portion whose Names are written in the Book of Death, and blotted by God out of the Book of LIFE, though they hope to find them written there.
```

Comment:
Hmmm.... I think better than an `[.old-style]` here would have been to have made the big long paragraph part a [.section-summary-preface].  Let's switch it to that.  And then, italicize the uppercase words as described in the previous comment as well.

Reply:
Okay, I didn't realize I could break that off, and I didn't think I should use the [.section-summary-preface] for the whole thing since it was a sub-heading itself. But I see what you're saying here!